daemon/execdriver/native/template/default_template_linux.go

```diff
@@ -94,5 +97,10 @@ func New() *configs.Config {
 		container.AppArmorProfile = "docker-default"
 	}
 
+	if SystemdCgroups {
+		container.Cgroups.Parent = "system.slice"
```

Comment:
Now that system.slice is defined as the default systemd parent, it should be used in Apply (L166) and getSubsystemPath (L408) instead of defining default slice values there.